secator/tasks/fping.py

```
@@ -17,17 +17,24 @@ class fping(ReconIp):
 		DELAY: 'period',
 		PROXY: OPT_NOT_SUPPORTED,
 		RATE_LIMIT: OPT_NOT_SUPPORTED,
 		RETRIES: 'retry',
 		TIMEOUT: 'timeout',
 		THREADS: OPT_NOT_SUPPORTED,
-		'reverse_dns': 'r'
+		'count': 'c',
+		'show_name': 'n',
+		'use_dns': 'd',
+		'summary': 's',
 	}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)